src/lib/routes/auth/reset-password-controller.ts

```diff
@@ -56,6 +56,9 @@ class ResetPasswordController extends Controller {
             permission: NONE,
             middleware: [
                 openApiService.validPath({
+                    summary: 'Validates a token',
+                    description:
+                        'If the token is valid returns the user that owns the token',
```

Comment:
What does it return otherwise? A 401? 

```suggestion
                        'Validates a user token. If the token is valid, returns the user that owns the token. Otherwise ...',
```